scripts/base/packet-protocols/gre/main.zeek

```diff
@@ -6,5 +6,30 @@ export {
 
 event zeek_init() &priority=20
 	{
-	PacketAnalyzer::register_packet_analyzer(PacketAnalyzer::ANALYZER_GRE, 0x0069, PacketAnalyzer::ANALYZER_IEEE802_11);
+	# ARUBA
```

Comment:
Conceptually, I would have expected the other GRE 0 types (ERSPAN etc.) to appear here as well.